.ibm/pipelines/utils.sh

```diff
@@ -56,7 +56,27 @@ droute_send() {
       ARTIFACTS_URL="https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/pr-logs/pull/${REPO_OWNER}_${REPO_NAME}/${PULL_NUMBER}/${JOB_NAME}/${BUILD_ID}/artifacts/e2e-tests/${REPO_OWNER}-${REPO_NAME}/artifacts/${project}"
     else
       JOB_URL="${JOB_BASE_URL}/logs/${JOB_NAME}/${BUILD_ID}"
-      ARTIFACTS_URL="https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/${JOB_NAME}/${BUILD_ID}/artifacts/${JOB_NAME##periodic-ci-janus-idp-backstage-showcase-main-}/${REPO_OWNER}-${REPO_NAME}/artifacts/${project}"
+      # OpenShift CI artifact storage URL cannot be parametrized and need to be updated manually, same as `report_template` at https://github.com/openshift/release/tree/master/ci-operator/jobs/janus-idp/backstage-showcase
+      case "$JOB_NAME" in
+        "periodic-ci-janus-idp-backstage-showcase-1.2.x-e2e-tests-nightly")
```

Comment:
@zdrapela having a function that takes job name and generates and return full url would be a good idea instead of putting each job URL.

Reply:
or you can just do ARTIFACTS_URL="https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/test-platform-results/logs/${JOB_NAME}/${BUILD_ID}/artifacts/e2e-tests-nightly/janus-idp-backstage-showcase-nightly/artifacts/${project}"

Reply:
Yes, that would be great, but the artifact URL seems to be dependent also on the folder structure here: https://github.com/openshift/release/tree/master/ci-operator/step-registry/janus-idp/backstage-showcase
The challenge is how to parametrize the last part of the URL - for example `janus-idp-backstage-showcase-helm-aks-nightly/artifacts/${project}"` 
The `janus-idp-backstage-showcase-helm-aks-nightly` is not a job name and I couldn't find any [variable](https://github.com/openshift/test-infra/blob/master/prow/jobs.md) that caries this.

Reply:
It would be good to separate `droute_send` into smaller pieces, and I'll work on that.